docs/Fields.md

```diff
@@ -26,7 +26,9 @@ export const PostList = (props) => (
 All field components accept the following attributes:
 
 * `source`: Property name of your entity to view/edit. This attribute is required.
-* `label`: Used as a table header of an input label. Defaults to the `source` when omitted.
+* `label`: In a datagrid, used as a table header of an input label.
```

Comment:
please undo this change. You're talking about specifics in a place where the reader expects general information.